handlers/soa_resource.go

```diff
@@ -64,9 +64,9 @@ func HandleCreateStatementOfAffairs(svc dao.Service, helperService utils.HelperS
 		}
 
 		// Validate the supplied attachment is a valid type
-		if attachment.Type != "statement-of-affairs-director" && attachment.Type != "statement-of-affairs-liquidator" {
+		if attachment.Type != "statement-of-affairs-director" && attachment.Type != "statement-of-affairs-liquidator"  && attachment.Type != "statement-of-concurrence" {
```

Comment:
in this modified code you have not used the constants for "statement-of-affairs-director"?

Reply:
done